ModuleConfig.cfc

```diff
@@ -11,7 +11,8 @@ component {
 
     function onLoad() {
         var resourceService = wirebox.getInstance( "ResourceService@cbi18n" );
-        resourceService.loadBundle( "#moduleMapping#/resources/i18n/coda" );
+        resourceService.loadBundle( "#moduleMapping#/resources/i18n/coda","en_US" );
+        resourceService.loadBundle( "#moduleMapping#/resources/i18n/coda","de_DE" );
```

Comment:
From how I understand cbi18n should work is that

`resourceService.loadBundle( "#moduleMapping#/resources/i18n/coda" );` would load all bundles following the naming convention. Didn't pick up `de_DE` for me, explicit loading as in my change does.

Reply:
I'll check with the cbi18n authors on what we need to do here to be a bit more by convention.